Shoko.Server/Settings/ImportSettings.cs

```diff
@@ -8,7 +8,7 @@ public class ImportSettings
         
         public List<string> VideoExtensions { get; set; } = new List<string> { "MKV", "AVI", "MP4", "MOV", "OGM", "WMV", "MPG", "MPEG", "MK3D", "M4V" };
 
-        public List<string> Exclude { get; set; } = new List<string> { "$RECYCLE.BIN", ".Recycle.Bin", ".Trash-" };
+        public List<string> Exclude { get; set; } = new List<string> { @"(\\|\/)\$RECYCLE\.BIN(\\|\/)", @"(\\|\/)\.Recycle\.Bin(\\|\/)" };
```

Comment:
You'll want to leave `.Trash-` in there, as it's a trash folder that ubuntu uses, alongside other OS's

https://askubuntu.com/questions/282332/what-is-the-trash-1000-folder-and-how-to-delete-it

You'd want the regex: `[\\\/]\.Trash-\d+[\\\/]` (I am using [] instead of a capture group by habit)

Reply:
Added it back in :)